test/e2e/providers/aws/aws.go

```diff
@@ -146,8 +145,9 @@ func getLatestWindowsAMI(ec2Client *ec2.EC2, hasCustomVXLANPort bool) (string, e
 	}
 
 	// Find the last created image
-	latestImage := describedImages.Images[0]
-	latestTime, err := time.Parse(time.RFC3339, *latestImage.CreationDate)
+	latestImage := describedImages.Images[2]
+	// WINC-833 temp fix. We don't need comp logic at present.
+	/*latestTime, err := time.Parse(time.RFC3339, *latestImage.CreationDate)
```

Comment:
without this logic this will be picking a random image from the list wouldnt it?
potentially being the broken image?

Reply:
As per the comment, it says `// Find the last created image`. It is stored in 0th index. So I assumed it is already sorted. Hence I am picking up from 2nd index.

Reply:
My understanding is the list is not sorted, otherwise we wouldn't have needed the logic being commented out. 
`Find the last created image` would be referring to this entire code section